src/test/java/com/marathon/hazelcast/servicediscovery/DiscoveryTest.java

```diff
@@ -84,17 +84,28 @@ public void testMultiMemberDiscovery() throws UnknownHostException, InterruptedE
                         .withHeader("Content-Type", "application/json")
                         .withBody(mapper.writeValueAsBytes(response))));
 
-        HazelcastInstance hazelcast1 = getHazelcastInstance(5701);
-        HazelcastInstance hazelcast2 = getHazelcastInstance(5702);
-        HazelcastInstance hazelcast3 = getHazelcastInstance(5703);
+        HazelcastInstance hazelcast1 = getHazelcastInstanceWithoutBasicAuth(5701);
```

Comment:
Let's have test cases for both with and without auth

Reply:
yes, if you see here 87 and 88 line one's are Without Auth instances and 89th line one is With Auth. And in 91st line asserting that cluster members should be 3